conf/test.config

```diff
@@ -20,10 +20,21 @@ params {
     max_time   = '6.h'
 
     // Input data
-    input = "${baseDir}/assets/samplesheet.csv"
+    input = "${projectDir}/assets/samplesheet.csv"
 
     // Genome references
     fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/nascent/reference/GRCh38_chr21.fa'
     gtf = 's3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.gtf'
+
+    assay_type = "GROseq"
+    skip_grohmm = true
     tuning_file = "${projectDir}/tests/subworkflows/local/grohmm/tuningparams_small.csv"
+    filter_bed = "${projectDir}/tests/samplesheets/region.bed"
```

Comment:
Is this a file that would be used for all users, or could it be moved to test-datasets given this is in the `test.config`?

Reply:
Yes, but it'll also work from here, and that way it's quick to update.